src/System.Diagnostics.StackTrace/src/System.Diagnostics.StackTrace.csproj

```diff
@@ -27,6 +32,18 @@
   <ItemGroup>
     <Compile Include="System\Diagnostics\StackFrameExtensions.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)' == ''">  
+    <Compile Include="System\Diagnostics\StackTraceSymbols.CoreCLR.cs" />  
+    <ProjectReference Include="..\..\System.Collections\src\System.Collections.csproj" />  
+    <ProjectReference Include="..\..\System.Threading.Overlapped\src\System.Threading.Overlapped.csproj" />  
```

Comment:
Sorry, I don’t understand exactly. Are you saying that I need to exclude building StackTraceSymbols.CoreCLR.cs for the Windows_NT with a conditional like ‘$(OSGroup)’ != ‘Windows_NT’?

We want this code on Windows too.